pkg/scheduler/cache/refermetrics/metrics_client.go

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Suggest to use a formal directory name instead of `refermetrics`.

Reply:
@william-wang Solved. I have moved it to `metrics/source`